File: Com.Fubei.Api.OpenApi/Biz/AgentApi/Model/Request/AOrderQueryParam.cs
using System;
using System.Collections.Generic;
using System.Text;
using Com.Illuminati.Galileo.Model;
using Newtonsoft.Json;

namespace Com.Fubei.Api.OpenApi.Biz.AgentApi.Model.Request
{
    public class AOrderQueryParam : BaseEntity
    {
        [JsonProperty("merchant_id")]
        public int? MerchantId { get; set; }

        [JsonProperty("order_sn")]
        public string OrderSn { get; set; }

        [JsonProperty("merchant_order_sn")]
        public string MerchantOrderSn { get; set; }

        [JsonProperty("ins_order_sn")]
        public string InsOrderSn { get; set; }
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/AgentApi/Model/Request/AOrderQueryParam.vb
Imports Com.Illuminati.Galileo.Model
Imports Newtonsoft.Json

Namespace Biz.V2.Model.Request
    
    ' 订单查询
    ' http://docs.51fubei.com/agent-api/payment/orderQuery.html
    Public Class AOrderQueryParam
        Inherits BaseEntity

        <JsonProperty("merchant_id")>
        Public Property MerchantId As Integer?

        <JsonProperty("order_sn")>
        Public Property OrderSn As String

        <JsonProperty("merchant_order_sn")>
        Public Property MerchantOrderSn As String

        <JsonProperty("ins_order_sn")>
        Public Property InsOrderSn As String
    End Class
End Namespace
